images/ubuntu/editor/Dockerfile

```diff
@@ -24,6 +24,14 @@ RUN for mod in $module; do \
 	# Set execute permissions for modules
 	&& chmod -R 755 /opt/unity/editors/$version/Editor/Data/PlaybackEngines
 
+RUN echo "$version-$module" | grep -q -v '^\(2021.2\|2021.3\|2022\).*linux' \
+  && exit 0 \
+  || unity-hub install-modules --version "$version" --module "linux-server" --childModules | tee /var/log/install-module-${linux-server}.log && grep 'Missing module' /var/log/install-module-${linux-server}.log | exit $(wc -l);
+
+RUN echo "$version-$module" | grep -q -v '^\(2021.2\|2021.3\|2022\).*windows' \
+  && exit 0 \
+  || unity-hub install-modules --version "$version" --module "windows-server" --childModules | tee /var/log/install-module-${windows-server}.log && grep 'Missing module' /var/log/install-module-${windows-server}.log | exit $(wc -l);
```

Comment:
Looking at this one last time before merging.

I don't think ${linux-server} and ${windows-server} are defined. Should it be 

`/var/log/install-module-windows-server.log` instead of `/var/log/install-module-${windows-server}.log`?

Reply:
fixed